moveit_core/constraint_samplers/src/constraint_sampler_manager.cpp

```diff
@@ -61,16 +63,16 @@ constraint_samplers::ConstraintSamplerManager::selectDefaultSampler(const planni
   if (!jmg)
     return constraint_samplers::ConstraintSamplerPtr();
   std::stringstream ss;
-  ss << constr;
-  ROS_DEBUG_NAMED("constraint_samplers",
+  ss << constr.name;
```

Comment:
Isn't there a way to print the whole message as a string anymore?

Reply:
I still think that the full constraint should be printed here instead of only the name.

Reply:
There is no such implementation for ROS2. I have asked that on the following issue : https://github.com/ros2/common_interfaces/issues/69#event-2410934891 redirected to https://github.com/ros2/rosidl/issues/259